packages/svelte/src/internal/client/dev/inspect.js

```diff
@@ -33,8 +33,9 @@ export function inspect(get_value, inspector, show_stack = false) {
 				inspector(...snap);
 
 				if (!initial) {
+					const stack = get_stack('$inspect(...)');
 					// eslint-disable-next-line no-console
-					console.log(get_stack('UpdatedAt'));
+					if (stack) console.log(stack);
```

Comment:
we could do this...

```suggestion

					if (stack) {
						// eslint-disable-next-line no-console
						console.groupCollapsed('stack trace');
						// eslint-disable-next-line no-console
						console.log(stack);
						// eslint-disable-next-line no-console
						console.groupEnd();
					}
```

...and it would look like this:

<img width="1297" height="667" alt="image" src="https://github.com/user-attachments/assets/b9d53077-646b-450d-8bb1-ffb19ff56b97" />

More compact (especially when the stack trace is large), though it hides the information behind a click. Maybe that's the right trade-off?

Reply:
The real question is what does it look like in Firefox?